lib/reesmat.gi

```diff
@@ -246,8 +246,8 @@ function(S, mat)
   
   for x in mat do 
     if ForAny(x, s-> not s in S) then
-      Error("usage: the entries of <mat> must belong to <S>,");
-      return;
+      ErrorNoReturn("the entries of the second argument (a rectangular ", 
+                    "table) must belong to the first argument (a semigroup)");
```

Comment:
This would be a really useful change, then the error messages in this PR could be shorter, and others else where would be comprehensible without consulting the source code. I'm all for this.